tools/slang-unit-test/unit-test-coverage-cpu-runtime.cpp

```
@@ -0,0 +1,348 @@
+    SLANG_CHECK_ABORT(counterCount > 0);
+
+    SLANG_CHECK_ABORT(syntheticResources->getResourceCount() == 1);
+    slang::SyntheticResourceInfo resourceInfo;
+    SLANG_CHECK_ABORT(syntheticResources->getResourceInfo(0, &resourceInfo) == SLANG_OK);
+    SLANG_CHECK(resourceInfo.uniformOffset >= 0);
+    // The CPU representation of the coverage buffer is a
+    // (data pointer, element count) pair; the reported stride is the size of
+    // that representation in the global-params payload.
+    SLANG_CHECK(resourceInfo.uniformStride == int32_t(sizeof(CpuStructuredBufferView)));
+
+    // The kernel executes in-process, so host and kernel agree on pointer
```

Cppcheck (2.21.0)

[style] The function 'hashValue' is never used.

(unusedFunction)